### ISSUE TITLE
Docker image crashlooping / `This script requires Python 3.5+`

### ISSUE THREAD

**Issue by tuxpeople** [timothymiller/cloudflare-ddns#65]
**Describe the bug**
Image crashlooping

```                                                                                                                                                                                                     
 Traceback (most recent call last):                                                                                                                                                                                                       
   File "/cloudflare-ddns.py", line 192, in <module>                                                                                                                                                                                      
     raise Exception("🐍 This script requires Python 3.5+")                                                                                                                                                                               
 Exception: 🐍 This script requires Python 3.5+                                                                                                                                                                                           
```

**To Reproduce**
Update to latest docker hub image with `digest: sha256:8bd682a383014dfea17c8e2d32bac59e3a1e5a0e767cb737469056cba5d4bbac`

I can't tell for sure if the error just came with this release, or if it exists longer.

**Expected behavior**
Image running

**Additional context**
Using it in a `k3s` Kubernetes cluster.

**Comment by rojosinalma** [comment]
#66

**Comment by timothymiller** [comment]
Fixed with @rojosinalma’s PR #66
